Gemfile

```diff
@@ -25,6 +25,11 @@ else
   gem 'rake', '~> 10.0' # rake 11 requires Ruby 1.9.3 or later
 end
 
+# Version 3 of mime-types 3 requires Ruby 2.0
+if RUBY_VERSION < '2.0.0'
+  gem 'mime-types', '< 3'
+end
```

Comment:
https://travis-ci.org/rspec/rspec-rails/builds/118239747 failed to bundle because [v2.6.4 of `mail` is out](https://rubygems.org/gems/mail/versions/2.6.4), which allows bundling with `mime-types < 4` instead of, as earlier, `mime-types < 3`.